test/check-subscriptions

```diff
@@ -380,7 +380,8 @@ class TestSubscriptions(SubscriptionsCase):
 
         dialog_register_button_sel = "footer .pf-m-primary"
         b.click(dialog_register_button_sel)
-        b.wait_not_present(dialog_register_button_sel)
+        with b.wait_timeout(360):
+            b.wait_not_present(dialog_register_button_sel)
```

Comment:
hmm the code here is the same as used in other tests, and in this case the connection to insights is disabled when registering; how come this is needed?

Reply:
Hmm, maybe registering was actually the problem, not connecting to insights. I'll check this more carefully by comparing old and new images.

Reply:
Note that this didn't "binary" break on an image refresh. It has been flaky for months, and just recently got bad enough that it pretty much always fails. But there's no "known good state" to compare it to.

Reply:
Alright, without the increased timeout, we get this warning:

```
WARNING: Waiting for !ph_is_present("footer .pf-m-primary") took 10.8 seconds, which is 72% of the timeout.
```

That's probably why I made it longer.

There is a similar warning for `testRegister`:

```
WARNING: Waiting for !ph_in_text("body","User doc is member of more organizations, but no organization was selected") took 7.7 seconds, which is 51% of the timeout.
```

So registration seems to brush up against the timeout in general.  I'll make these two places wait longer, in a separate commit.

Reply:
Done.